src/lava/proc/conv/models.py

```diff
@@ -27,14 +27,24 @@ class AbstractPyConvModel(PyLoihiProcessModel):
     dilation: np.ndarray = LavaPyType(np.ndarray, np.int8, precision=8)
     groups: np.ndarray = LavaPyType(np.ndarray, np.int8, precision=8)
 
+    def __init__(self) -> None:
+        super().__init__()
+        self.a_buf = None
```

Comment:
Could a_buf be defined in the same way as a_out? We may not need an explicit __init__() method then, but if it is needed we should define member variables consistently in the same place.

Reply:
We can define a_buf as a Var. I don't see it being useful though. It's just a numpy array here.

My understanding was that we had the design decision to define the ProcessModel vars and IO ports as static member to be efficient with memory. Static member don't work like in this case when a_buf is a simple numpy type, not LavaPyType. Hence, the need for __init__()

Reply:
I've changed a_buf to be a Var.